rootfs/bin/gh-dl-release

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
```

Comment:
It pains me we keep copying this into repos. Can't we just create our own `gh-dl-release` repo, and `ADD` the GitHub raw URL?